COMPATIBILITY.md

```diff
@@ -47,6 +47,7 @@ Newly filed issues should bear the label `pipeline` for ease of tracking.
 - [X] `HipChatNotifier` (`hipchat`): supported as of 1.0.0
 - [ ] `IronMQNotifier` (`ironmq-notifier`): [JENKINS-35505](https://issues.jenkins-ci.org/browse/JENKINS-35505)
 - [X] `LogParserPublisher` (`log-parser`): supported as of 2.0
+- [X] `RocketChatNotifier` (`rocketchatnotifier`): supported as of 0.5.0
```

Comment:
Still incorrect as previously noted. This is a custom step (`rocketSend`). The original `RocketChatNotifier` post-build action is not compatible with Pipeline AFAICT.